src/System.Text.Json/src/System/Text/Json/Serialization/Converters/DefaultEnumerableConverter.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Text.Json.Serialization.Policies;
+
+namespace System.Text.Json.Serialization.Converters
+{
+    internal class JsonEnumerableT<T> : ICollection<T>, IEnumerable<T>, IList<T>, IReadOnlyCollection<T>, IReadOnlyList<T>
```

Comment:
On naming: I don't think having a "T" suffix is standard. Just use `JsonEnumerable<T>`

Reply:
I added the "T" suffix so that the non-T-suffix names are available for the non-generic interface collection types that will be added later. I'll leave this per our offline discussion.

Reply:
Can't we have both `JsonEnumerable` and `JsonEnumerable<T>`?